use-timescale/regions.md

```diff
@@ -23,3 +28,11 @@ Timescale is available in these Amazon Web Services (AWS) regions:
 | `us-east-2`      | United States | Ohio           |
 | `us-west-2`      | United States | Oregon         |
 
+<Highlight type="cloud"  header="Need to deploy on Azure or Google Cloud Platform?">
+
+Use [Managed Services for Timescale][aiven-client-mst] to run TimescaleDB on Azure and 
+GCP for you in partnership with Aiven.
```

Comment:
This sentence reads a bit weird. Maybe let's just remove the Aiven bit.
"Use Managed Service for TimescaleDB (MST) to run TimescaleDB on Azure and GCP."